src/main/java/uk/gov/hmcts/reform/civil/model/Bundle.java

```diff
@@ -114,6 +127,18 @@ public String getFilename() {
         return filename;
     }
 
+    public Optional<LocalDateTime> getCreatedOn() {
+        return checkIsOptional(createdOn);
+    }
+
+    public Optional<LocalDate> getBundleHearingDate() {
+        return bundleHearingDate;
+    }
+
+    public Optional<String> getStitchingFailureMessage() {
+        return stitchingFailureMessage;
```

Comment:
Presumably this also will need `checkIsOptional` if its reason is that jackson may initialize `Optional` fields to null.